jruby/neo4j/driver/ext.rb

```diff
@@ -0,0 +1,6 @@
+module Neo4j
+  module Driver
```

Comment:
A patch/workaround for Jruby version < 9.2.8.0 where the zeitwerk is looking for a file named "jruby/neo4j/driver/ext.rb" to be present and with definition of ext module.